Expose grid and constellation settings

Carifio24 opened the pull request:
This PR exposes grid and constellation settings from WWT - in particular, the ability to toggle whether constellation properties, constellation boundaries, and several grid types are visible. These toggles are exposing as options in the viewer state, with the updated options widget UI as below:

![Screenshot from 2023-06-27 12-23-28](https://github.com/glue-viz/glue-wwt/assets/14281631/c8b92ef9-4eea-4b54-92f2-926b55bb4be0)

Carifio24 commented:
Only Python 3.7 is failing with unrelated errors, which it looks like we're planning to drop in #94 anyways.

astrofrog reviewed (commented):
This looks good to me so far! One question I have is whether there is any option to show the coordinates on the grids in WWT and whether this is something we can expose here?

Carifio24 commented:
Showing grid coordinates (for all of the grids) is implemented in the web engine but not yet exposed to pywwt. I'm working on exposing the ability to change grid colors; I'll add grid text toggling as well

Carifio24 commented:
Marking this as draft as it anticipates the changes from https://github.com/WorldWideTelescope/pywwt/pull/354 and https://github.com/WorldWideTelescope/pywwt/pull/355, which aren't yet released but will be soon.

Carifio24 commented:
All of these features have now landed in `pywwt`, so this should be ready to review.

@astrofrog I've added relevant UI updates to the Jupyter viewer as well. There are a lot of options now, so I set up the layout widget as an accordion whose sections match the tabs in the Qt version. I've also added a pin to the current `pywwt` version.

astrofrog reviewed (approved):
Looks great, thank you for all the work on this!

astrofrog commented:
Regarding the CI failure I think we should bump the minimum Python to 3.8, but can be done in a separate PR (@dhomeier could you take a look at updating this?)
